### PR TITLE
Feature/249 create message base

### PR DESCRIPTION
**Pull request by todd-dembrey**

### CODE REVIEW

**Comment by todd-dembrey** [comment]
I will need to add some tests for the messaging logic, but generally it looks to be working. 

@zerolab  review please

File: opentech/apply/determinations/tests/test_views.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def test_can_edit_draft_determination(self):
         }, 'form')
         self.assertContains(response, '[Draft] Approved')
         self.assertContains(response, reverse(self.url_name.format('form'), kwargs=self.get_kwargs(submission)))
-        self.assertContains(response, 'Accepted determination draft message')
+        self.assertNotContains(response, 'Accepted determination draft message')
```
Comment (todd-dembrey) [inline opentech/apply/determinations/tests/test_views.py]: @zerolab Your original code was creating an activity when the draft message was saving (see below) This shouldn't happen.
```
Sent a Needs more info determination for <title>: Accepted determination draft message
```

**Comment by todd-dembrey** [comment]
@chris-lawton Could you cast your eyes over the scss/html for the messages please

**Comment by zerolab** [comment]
LGTM. We discussed all outstanding questions on the taken approach. Nice one

File: opentech/static_src/src/sass/apply/components/_messages.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            border: solid 1px darken($color--pastel-green, 20%);
+        }
+
+        &--hide {
+            max-height: 0px;
+            padding-top: 0px;
+            padding-bottom: 0px;
+            border: 0 none;
+            transform-origin: top;
+            transition: 0.25s all ease-out;
```
Comment (chris-lawton) [inline opentech/static_src/src/sass/apply/components/_messages.scss]: `transition: all $transition`

**Pull request merged**